fakestorage/object.go

```diff
@@ -174,6 +180,48 @@ func (s *Server) deleteObject(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 }
 
+func (s *Server) listObjectACL(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	obj, err := s.GetObject(vars["bucketName"], vars["objectName"])
+
+	if err != nil {
+		http.Error(w, "not found", http.StatusNotFound)
+		return
+	}
+
+	response := newACLListResponse(obj)
+	json.NewEncoder(w).Encode(response)
+}
+
+func (s *Server) setObjectACL(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+
+	bodyMap := make(map[string]string)
+	defer r.Body.Close()
+	body, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		http.Error(w, "bad request", http.StatusBadRequest)
+		return
+	}
+
+	json.Unmarshal(body, &bodyMap)
```

Comment:
Please check the error here. Also, where is bodyMap being used?

Reply:
removed